### DISCUSSION TITLE
setAuth 写法是不是可以统一一下

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#707] [Ideas]
同时存在两种写法（加上 `setReplyAuth` 三种）感觉有点怪
<img width="830" alt="image" src="https://user-images.githubusercontent.com/9006264/157870133-f36ee195-63fb-43f4-85ba-d860a306c52d.png">

**Comment by Arondight** [comment]
这个我感觉还挺正常的，也是一种普遍的写法吧。最开始是只支持设置单一权限，插件写了个封装的，后来就把封装做到里面了。似乎 js  代码会普遍这么写，我实际上用起来感觉也挺好的（可以看一下开发手册中数据库的文档，有效治疗强迫症）

**Reply by Arondight** [reply]
那个封装似乎可以拆掉，统一不禁止管理员的权限，然后通知已经做到里面了，明天搞一搞
